.github/workflows/build-test-coverage.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Test & Coverage
+on:
+    pull_request:
+    push:
+
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.12"
+    - name: Run tests
+      run: |
+        pip install -q -U pip
+        pip install -q pipenv
+        pipenv install --dev
+        pipenv run pytest --cov-report=xml --cov=newrelic_lambda_cli tests
+    - name: Upload coverage report
+      run: bash -c "$(curl -s https://codecov.io/bash)"
+
+  code-style:
+    name: Code style
+    runs-on: ubuntu-latest
+    needs: test
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Check code style
+        run: |
+          pip install -q -U pip
+          pip install -q -U 'black<22.10.1'
+          black --check newrelic_lambda_cli tests
+      - name: Check README
+        run: |
+          pip install -q -U pip
+          pip install -q -U setuptools twine wheel
+          python setup.py sdist bdist_wheel
+          twine check dist/*
```

Comment:
This seems fine for now, but in the future if you'd like help setting up a better linting flow we could get Megalinter installed to automatically push formatting updates and run a variety of linters. Might be particularly helpful since we have non-python devs that occasionally touch this repo that might not want to use black.

Reply:
Experimented with adding [mega-linter](https://github.com/newrelic/newrelic-python-agent/blob/main/.github/workflows/mega-linter.yml) but it gave multiple changes for file names and others. Will incorporate it in other PR.